frontend/src/core/config/config-schema.ts

```diff
@@ -204,6 +204,7 @@ export const AppConfigSchema = z
     html_head_file: z.string().nullish(),
     auto_download: z.array(z.enum(["html", "markdown", "ipynb"])).default([]),
     sql_output: SqlOutputSchema,
+    file_stores: z.array(z.string()).optional(),
```

Comment:
rather than putting this in app config here, could we place it in the `mount(` function

Reply:
this way, you should be able to even pass the full store (not just a string)

Reply:
`__MARIMO_MOUNT_CONFIG__` works as well. the code i shared is not shipped yet, but will be in the next release